java/client/src/main/java/glide/api/commands/servermodules/GlideJson.java

```diff
@@ -0,0 +1,411 @@
+     * @return For JSONPath (path starts with <code>$</code>): Returns a stringifies JSON list of
+     *     bytes replies for every possible path, or a byte string representation of an empty array,
+     *     if path doesn't exist. For legacy path (path doesn't start with <code>$</code>): Returns a
+     *     byte string representation of the value in <code>path</code>. If <code>path</code> doesn't
+     *     exist, an error is raised. If <code>key</code> doesn't exist, returns null.
+     * @example
+     *     <pre>{@code
+     * String value = client.Json.set(client, "doc", "$").get();
+     * assert value.equals("{'a': 1.0, 'b': 2}");
+     * }</pre>
+     */
+    public static CompletableFuture<Object> get(BaseClient client, String key, String path) {
+        return executeCommand(client, new String[] {JSON_GET, key, path});
+    }
+
+    /**
+     * Retrieves the JSON value at the specified <code>path</code> stored at <code>key</code>.
+     *
+     * @param client The Valkey GLIDE client to execute the command.
+     * @param key The <code>key</code> of the JSON document.
+     * @param path The path within the JSON document.
+     * @return For JSONPath (path starts with <code>$</code>): Returns a stringifies JSON list of
+     *     bytes replies for every possible path, or a byte string representation of an empty array,
+     *     if path doesn't exist. For legacy path (path doesn't start with <code>$</code>): Returns a
+     *     byte string representation of the value in <code>path</code>. If <code>path</code> doesn't
+     *     exist, an error is raised. If <code>key</code> doesn't exist, returns null.
+     * @example
+     *     <pre>{@code
+     * GlideString value = client.Json.set(client, gs("doc"), gs("$")).get();
+     * assert value.equals(gs("{'a': 1.0, 'b': 2}"));
+     * }</pre>
+     */
+    public static CompletableFuture<Object> get(
+            BaseClient client, GlideString key, GlideString path) {
+        return executeCommand(client, new GlideString[] {gs(JSON_GET), key, path});
+    }
+
+    /**
+     * Retrieves the JSON value at the specified <code>paths</code> stored at <code>key</code>.
+     *
+     * @param client The Valkey GLIDE client to execute the command.
+     * @param key The <code>key</code> of the JSON document.
+     * @param paths List of paths within the JSON document.
+     * @return Returns a stringifies JSON object in bytes, in which each path is a key, and it's
+     *     corresponding value, is the value as if the path was executed in the command as a single
+     *     path. If <code>key</code> doesn't exist, returns null.
+     * @example
+     *     <pre>{@code
+     * String value = client.Json.set(client, "doc", new String[] {"$.a", "$.b"}).get();
+     * assert value.equals("{\"$.a\": [1.0], \"$.b\": [2]}");
+     * }</pre>
+     */
+    public static CompletableFuture<Object> get(BaseClient client, String key, String[] paths) {
```

Comment:
All get() function definitions are missing NonNull annotation for the function parameters.